ui/components/app/snaps/snap-ui-renderer/snap-ui-renderer.js

```diff
@@ -86,7 +86,7 @@ export const UI_MAPPING = {
     children: [mapToTemplate(props.value, elementKey)],
     props: {
       label: props.label,
-      variant: props.variant,
+      state: props.state,
```

Comment:
```suggestion
      state: props.variant,
```
Otherwise this is a breaking change to the snaps API that we don't wanna make right now.